### DISCUSSION TITLE
How to select text using keyboard without using mouse?

### DISCUSSION THREAD

**Discussion by x7z7** [ahrm/sioyek#769] [Q&A] [answered]
Currently, the only way of highlighting that I am aware of is left-clicking the text, dragging it, and then using `ha` (for example) to highlight the selected text. But I only find using the mouse for selecting the text whenever I use sioyek.

Is there a way to highlight text word-by-word (and also line-by-line) in sioyek without using the mouse?

**Comment by ahrm** [comment]
There is a `keyboard_select` command, which selects the text using labels shown next to the words.

**Reply by x7z7** [reply]
Yea, it can select words. But I'm not sure how I can "drag" it. If I go into command mode and type `keyboard_select` and pick a word to choose, it highlights the words I'm not sure how I can continue the select from the chosen word either left or right. Using arrow keys (left/right arrows) doesn't drag the selection.

**Reply by ahrm** [reply, marked as answer]
You can't drag, but you can select an interval by entering the labels corresponding to the begin and the end of selection (separated by a space).

**Reply by x7z7** [reply]
Thank you so much!
